src/purview.ts

```diff
@@ -1250,4 +1250,4 @@ export {
   PurviewEvent,
 } from "./types/ws"
 export { css, styledTag, CSS } from "./css"
-export * from "./types/jsx"
+export { JSXInternal as JSX }
```

Comment:
The namespace is exported as a utility. The previous `*` may not have been doing very much besides including the `declare global { }` syntax used there? Not sure. I think it's fine this way.